tqdm.py

```diff
@@ -0,0 +1,42 @@
+from time import sleep
+import datetime as dt
+import sys
+
+
+def Tqdm(iterable):
+    iteration_counter = 1
+    total_iterations = len(iterable)
+    start_time = dt.datetime.now()
+    for obj in iterable:
+        # number of iteration
+        sys.stdout.write('\rIteration {} of {}     '.format(iteration_counter, total_iterations))
+
+        # elapsed time
+        datetime_now = dt.datetime.now()
+        curr_duration = datetime_now - start_time
+        sys.stdout.write('elapsed time: {}     '.format(curr_duration))

```

Comment:
You can use Python formatting to add whitespaces (instead of adding them yourself).

Reply:
Done